### DISCUSSION TITLE
Stationarity is Dead

### DISCUSSION THREAD

**Discussion by eculler** [earthlab-education/Earth-Analytics-AY24#389] [Reading Responses]
Following up on our introduction of the Linear OLS Regression, the reading for the next two weeks is [*Stationarity is Dead: Whither Water Management* by Milly et al.](https://www.science.org/doi/10.1126/science.1151915) [Stationarity Is Dead: Whither Water Management? | Science. (n.d.). Retrieved September 13, 2023, from https://www.science.org/doi/10.1126/science.1151915]

## The stationarity assumption allows hydrologists and others to create simple models of how natural systems behave, by assuming they won't change over time.

Stationarity is an idea used heavily in classical statistics that all data points in a sequence have been drawn from the **same probability distribution**. Changes in both the mean and variance of streamflow and other values have been consistently observed in the last century, violating this notion and the models that use it as an assumption.

## Questions to get started with
1. Why are statistical models important to decision-making?
2. What are some potential real-world consequences of an invalid assumption of stationarity?
3. Can you think of any problems with abandoning the stationarity assumption entirely? For a longer take on this, check out the response article *Stationarity is undead*.

**Comment by annaposlednik** [comment]
As the well-known saying goes "all models are bad, some are useful." Statistical models are critical to decision-making because while they rely on a number of assumptions, and are only as good as the data you put in, they can find correlations between phenomena and forecast future conditions, as well as explain past/present conditions. Therefore, the results of statistical models could aid in management and policy, especially in regards to implications from climate change. However, invalid assumptions of stationarity could lead to very inaccurate model outputs that could be devastating for communities that could not properly forecast and prepare for a  climate scenario (like estimating flooding). In my field of ecology, statistical models are used regularly to understand ecosystem dynamics, and having invalid assumptions leads to poorly fitting models, which could have implications for local ecosystem management, restoration and conservation efforts. Abandoning assumptions of stationary though, requires more complex statistical modeling, which could be harder to interpret and therefore less useful (if you don't understand the output of your model, then you can't use it to make inferences).

**Comment by ZacharyCJDecker** [comment]
_1. Why are statistical models important to decision-making?_

While observations are the most direct determination, they cannot determine all variables over all times and in all places. Statistical models allow for one to fill in the gaps left by a lack of observation. These gaps can be critical to understanding the situation and therefore critical to decision-making. 

_2. What are some potential real-world consequences of an invalid assumption of stationarity?_

My understanding of the stationary assumption is that it allows for straight-forward variability estimates by assuming the amount of variability is defined. If this assumption is invalid then (in a worst case) the variability may be underpredicted. As such, "extreme" events will occur at a frequency greater than the model's variability would predict. Decision-makers and mangers will likely have not planned for such events if those models did not deem them probable. 

_3. Can you think of any problems with abandoning the stationarity assumption entirely? For a longer take on this, check out the response article Stationarity is undead._

Without reading the "stationary is undead" article in full, my thought is that abandoning the assumption entirely means one must start from scratch when determining the variability. One could do this, maybe robustly, if they had a reliable and lengthy time series of the desired variable which captured a statistically acceptable amount of variability. Yet, such time series are likely extremely rare. Plus, if stationary is truly invalid then the variability in the timeseries would not be representative of the future variability. In short, the future variability would need to also be modeled introducing additional uncertainty.

**Comment by hanried** [comment]
Statistical models are important to decision-making because they provide evidence, facts, and predictions to base a decision on. It’s also helpful that models can provide a clear summary of data; when making a decision, it’s nice to have a chunk of data synthesized into one model instead of having to look through a table of values or something like that. 

If an invalid assumption of stationarity is used, models could be outdated or naive because the models aren’t taking into account recent changes. As the article mentions, Earth’s climate is notably changing in ways that affect the water cycle. These changes could be outside of the “unchanging envelope of variability” that stationarity relies on so a model that assumes stationarity could be inaccurate and unreliable. Any decisions made based on such a model could also then be inaccurate and unreliable. 

One problem that comes to mind with abandoning the stationarity assumption is simply that it has been relied on for so long. Because of this, I think a transition away from stationarity would need to be carefully considered and planned so models that relied on stationarity could have some sort of instructions on how to be updated. Also, it seems like stationarity provides some control to some variables in water modeling. Without that control, would water models become much more difficult to create? Would we need some sort of new “control” to help reduce the things that have to be accounted for in water models?

**Comment by lauren-pease** [comment]
Statistical models can be leveraged to describe our expectations for large-scale and localized events and processes, and inform the planning of human infrastructure. The assumptions of a given data distribution play a role in model effectiveness. For instance, when describing the current hydroclimate of a region, the relevant data is likely reflective of a variability not yet understood. A hydraulic engineer may plan a few months ahead for a seasonal reservoir release cycle by relying on a model based on 100 years of historical data. If there is an especially wet or dry season that is triple above or below average precipitation, this season may confuse both short-term and long-term forecasts. 

That said, when a nonstationary model is employed to counter uncertainty and the model structure is undetermined, stationary models can be retained as sources of theoretically coherent understanding and used to guide management and solution design. Additionally, expanding the scope of the knowledge base assessed is crucial in domains like hydrology where the range of “normal” climate we have experienced is only a slice of our present epoch spanning thousands of years and periods of megadroughts and megafloods.

**Comment by gpb3037** [comment]
1. Why are statistical models important to decision-making?

Models provide the opportunity to say "what if" to describe various scenarios for planning purposes related to various geographies, specific infrastructure, cultural groups, or responsible agencies.

2. What are some potential real-world consequences of an invalid assumption of stationarity?

Existing models that have been relied upon for long term planning may relate to projects that have been baked in to their pro forma.  One example related to hydrology in Colorado would be the Gross Dam Expansion from Denver Water https://www.denverwater.org/grossreservoir/about-the-project/what-is-the-gross-reservoir-expansion-project.  This project when originally built projected future need in building a dam that could be added on to for increased retention - there has been some contention regarding this as being too extensive from those who did not understand the original plan, and in fact it may turn out to be planning for too little expansion based upon past models.

3. Can you think of any problems with abandoning the stationarity assumption entirely? For a longer take on this, check out the response article Stationarity is undead.

Stationarity and related modeling can be one ongoing reference resource, and other probabilistic may be another to examine for comparison.  Drawing upon another local example, there was rainfall and flooding in the front range of Colorado in September 2013 https://bouldercast.com/the-2013-boulder-flood-two-years-and-three-billion-dollars-later/#:~:text=When%20it%20was%20all%20said,September%20and%20wettest%20month%20ever generally recognized as a rare event in the 100 to 1000 year range.  Now that the event has happened within recorded memory, it will tend to influence other decision making for increased awareness and designs for flexibility in response and preparation.  More resources may be dedicated to improving runoff, with the deficit perhaps experienced by other projects within overall limited budget availability.  For those areas without rare event references it will be interesting to see if a blend of stationarity and probabilistic are compatible for planning and if existing plans can consider non-stationarity based scenarios.

**Comment by allenmoench** [comment]
Why are statistical models important to decision making?

Statistical models assist in understanding the probability of particular outcomes in a situation. For example, risk management. In the case of the hydrologic predictions mentioned in the article, probability density functions are being used to determine risk related to hydrologic infrastructure and investment. The pdf’s are used to estimate risks associated with extreme scenarios such as drought or flood which could damage infrastructure or impact water availability. Understanding the probability of these kinds of events occurring is critical in guiding decisions about allocation of funds to these projects, how infrastructure is built, and other considerations.

What are some potential real-world consequences of an invalid assumption of stationarity?

Making an invalid assumption of stationarity could lead to an over- or under- estimation of risk, or other faulty reasoning leading to bad decision making. In the example of hydrologic risk management, an invalid stationarity assumption could lead to underestimating the risk of an extreme event such as a flood. The underestimation could lead to a lack of preparation, which could lead to much more dramatic consequences when the flood did hit. This could include loss of the infrastructure and loss of investment, or even loss of life.

The risks of faulty stationarity assumptions seem particularly important in the context of natural disaster risk management, however they could be equally dangerous in other contexts. For example, they could also have dramatic consequences in the context of predicting economic patterns or the risk of disease spread. 

Can you think of any problems with abandoning the stationarity assumption entirely?

Abandoning the stationarity assumption would require the creation of a new method of statistical model creation. It might require more extensive measurement and observations to acquire accurate information that can be used to make predictions. All of this will require more money and time.  Making predictions in an environment where stationarity is an invalid assumption would require understanding the degree of variation within the ‘variability envelope’ surrounding the predictions. This in turn would require developing a greater understanding of what changes are occurring, and how they affect the statistical models in use today. Many older statistical models might become invalid or have to be grandfathered in. This could amount to a re-working of current models, decisions, or infrastructure that’s already in place, which could require a very large amount of work at a time when infrastructure also needs to expand.

**Comment by npwelsh** [comment]
1. 
	Statistical models are one of the best tools we have a predicting future outcomes.  A good statistical model can help people plan for predicted scenarios, including making policy based on model output or designing infrastructure off it's suggested outcomes.  It is also help fill in some of the gaps left by real world observations.
2.
	Consequences include underestimating the frequency of extremes for events.  For example, assuming that once in a hundred year storms will only have a 1 in 100 chance occurring every year, when in recent years there have been multiple hundred year storms in the same place , means not planning for those outcomes.  This could mean not building up the correct infrastructure, having  the necessary evacuation plan, or the appropriate emergency response to multiple of these events. 
3. 
The main problem is that models will become more complex, with more variables needed, and therefore more uncertain, especially if they are new and have not been tested against observed real world results. In this case, a preliminary stationarity model might be more appropriate.  Regardless, it is important to clearly and accurately communicate how the model was made and the assumptions that went into it.

**Comment by brglea** [comment]
Statistical models are important to decision making because they inform or are used to evaluate risks, evaluate investment in infrastructure, predict future events/ behavior/ trends based on past data , and more. That goes to say that these statistical models carry heavy weight in decisions on many topics which have far reaching effects as they impact populations and environments around the world in different ways. These statistical models were previously assumed to be helpful after taking into account human disturbances; however, with the invalid assumption of stationarity (that natural change was small enough to allow for stationarity assumption), comes consequences. 

Some potential real-world consequences could be: inaccurate risk calculations resulting in potential collapse or failure of 'xyz', investment in infrastructure that won't be able to keep up with needs / or fails in some way, inability to account for large changes to extremes such as flood frequency. 

(An aside: The article didn't explicitly talk about this but tied to all of these consequences is money - if something collapses/ fails/ etc. who pays for replacement , where does that money come from, and what mechanisms and processes come forward in the aftermath. I know there is also a stationarity assumption used in time series models in economics (it's been a while so I hope I'm remembering correctly). I am curious if this study or research was expanded to be inter-disciplinary (or thinking in terms of interconnected systems) because I would assume that the stationarity assumption in economics related models would also be affected by climate change and the failure to accurately account for natural change? What, if anything, have other disciplines done about the stationarity assumption in their statistical models? )

So, if stationarity is dead, then what? If stationarity assumption is abandoned entirely, there may be problems with increased complexity, possible misuse of software, introducing an additional source of uncertainty that needs to be accounted for. At this point in time it doesn't seem like there is a clear cut solution to the invalid assumption of stationarity, but rather a possible nonstationarity route that carries its own issues. As always, it's important to document and clearly communicate what model you are using, what assumptions come with it, why you chose that model, and acknowledging that results may be physically inconsistent.

**Comment by ZimmZone** [comment]
**Why are statistical models important to decision-making?**

Statistical models provide a framework upon which decision can be made. They include a base level of knowledge garnered from past observations which allow decision-makers to make the best 'educated guesses' about the future possible. 

**What are some potential real-world consequences of an invalid assumption of stationarity?**

Stationarity, and previous decision based upon models using stationarity, are being invalidated by the consequences of climate change. This means that water management, such as levy's or dams, which were build upon these assumptions may no longer meet the future needs. For example, levy's build to withstand levels of storm surge may not be able to cope with the larger storm surges caused by climate change. 


**Can you think of any problems with abandoning the stationarity assumption entirely? For a longer take on this, check out the response article Stationarity is undead.**

Stationarity bases future circumstances upon past evidence and still serve as the bedrock from which other models or assumptions can be built. The model may be increasingly inaccurate but will need to be replaced or 'augmented' instead of simply abandoned. As we learn more about its weaknesses these can be addressed.

**Comment by amarangunic** [comment]
1. Why are statistical models important to decision-making?
This seasonal models are statistical tools used to analyze and forecast time series that show patterns that repeat at regular intervals, such as months, quarters or years. These models are especially useful for data showing seasonal variations, these are essential tools that facilitate better decision-making by providing insights,  also reducing uncertainties, and enabling organizations to make strategic choices based on evidence and analysis. As are the American model of the @noaa based on the long-term CFS model and the Copernicus, on the European ECMWF model. Although NOAA (U.S. National Oceanic and Atmospheric Administration) is not a seasonal model per se, but an agency that uses various models, including seasonal models, to study and forecast weather and climate phenomena.

2. What are some potential real-world consequences of an invalid assumption of stationarity?
Incorrectly assuming that a time series is stationary without considering seasonality can have serious consequences on forecast accuracy, resource management, and decision making in various areas. Proper analysis is critical to identify seasonal patterns and adjust models accordingly. 

The potential consequences of an invalid assumption of seasonality in a stationary model can be forecast errors, wrong decisions, inadequate planning, mismanagement, limitations in data analysis, long-term consequences, among others. 

3. Can you think of any problems with abandoning the stationarity assumption entirely? For a longer take on this, check out the response article Stationarity is undead.
The conclusion on the issue of stationarity in time series analysis, especially in light of the “Stationarity is Undead” article, is that stationarity should not be viewed as an essential requirement in time series analysis. Instead, analysts should be aware of the characteristics of the data and select models that adequately reflect the observed reality. This can lead to better decisions and more accurate forecasts in practice.
The assumption of stationarity, while common in time series analysis, can be misleading and limit the ability of analysts to capture the complexity of real-world data. Recognizing non-stationarity is essential, it is good to adopt flexible approaches, contextualize the analysis in context, and improve decision making.

**Comment by BioNCphilic** [comment]
Hey - just getting caught up from work travel and hurricane Helene. 
It's hard for me to respond to this right now because the city closest to us - Asheville - is working to restore 1800+ miles of water distribution lines right now. I have a spring on our property which has been resilient through drought and flood. But this article hits close to home even though I can't read the entire article.

**Questions**

1. Why are statistical models important to decision-making? - Models are a tool for making predictions so that resources can be allocated efficiently. 

3. What are some potential real-world consequences of an invalid assumption of stationarity? - The increase in severity of storms overwhelms the boundaries that are assumed by stationarity. When 100 year floods happen every decade, expectations for meeting this challenge are inadequate.

4. Can you think of any problems with abandoning the stationarity assumption entirely? For a longer take on this, check out the response article Stationarity is undead. - It seems that there are other uncertainties that complicate the issue. 

Again - my apologies. I'm finding it hard to focus.
